pdns/dnsdistdist/docs/reference/dq.rst

```diff
@@ -84,6 +84,48 @@ It also supports the following methods:
 
   :returns: true if the DO bit was set, false otherwise
 
+.. classmethod:: DNSQuestion.getTag(key) -> string
```

Comment:
Shouldn't this be `DNSQuestion:getTag(key)` (also for the other newly added functions?)